io.github.dfandrich.gpscorrelate.json

```diff
@@ -0,0 +1,63 @@
+{
+  "app-id":"io.github.dfandrich.gpscorrelate",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"gpscorrelate-gui",
+  "finish-args":[
+    "--socket=fallback-x11",
+    "--socket=wayland",
+	"--filesystem=home",
```

Comment:
Please re-indent this; also, what does it need home access for?

Reply:
I wasn't seeing any files from the chooser without --filesystem=home. But I realize now that there must be something else going on there. The code is using gtk_file_chooser_dialog_new which I assume should be working.

Reply:
You need to use a native_file_chooser to use the portals.